.github/workflows/release.yml

```diff
@@ -9,6 +9,21 @@ on:
         # and https://docs.npmjs.com/about-semantic-versioning
 
 jobs:
+  # Make sure eslint passes
+  eslint:
+    name: ESLint Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2.4.1
+        with:
+          node-version: '14'
+          cache: 'npm'
+      - name: Install dependencies and run eslint
+        run: |
+          npm install
+          npm run eslint
```

Comment:
Let's integrate this on line 54 below, before the call to `npm test` so that our dependency between release and build holds.

Reply:
done!